src/app/screens/connectors/ConnectGaloy/index.tsx

```diff
@@ -1,20 +1,24 @@
 import ConnectorForm from "@components/ConnectorForm";
 import Input from "@components/form/Input";
+import ConnectionErrorToast from "@components/toasts/ConnectionErrorToast";
 import axios from "axios";
 import { useState } from "react";
+import { useTranslation, Trans } from "react-i18next";
 import { useNavigate } from "react-router-dom";
 import { toast } from "react-toastify";
 import utils from "~/common/lib/utils";
 
 export const galoyUrls = {
   "galoy-bitcoin-beach": {
+    i18nprefix: "bitcoin_beach",
```

Comment:
Should we stick to snakeCase and use `i18nPrefix`?

Reply:
Oops, yes!